stable/insights-admission/templates/cert.yaml

```diff
@@ -3,52 +3,21 @@ apiVersion: cert-manager.io/v1alpha2
 kind: Certificate
 metadata:
   name: {{ include "insights-admission.fullname" . }}-cert
-  annotations:
-    "helm.sh/hook": pre-install
-    "helm.sh/hook-weight": "6"
 spec:
   commonName: {{ include "insights-admission.fullname" .}}.{{ .Release.Namespace}}.svc
   dnsNames:
+  - {{ include "insights-admission.fullname" . }}.{{ .Release.Namespace }}
   - {{ include "insights-admission.fullname" . }}.{{ .Release.Namespace }}.svc
   - {{ include "insights-admission.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
it will be nice to refactor `cluster.local` to values. I have seen couple cluster change this to something else

Reply:
Ooh good to know, thanks!